Dockerfile.build.template

```diff
@@ -0,0 +1,37 @@
+# Build nodejs dependencies
+FROM resin/%%ARCH%%-node:6.2-slim
+
+WORKDIR /usr/src/app
+
+RUN apt-get update \
+	&& apt-get install -y \
+		g++ \
+		libsqlite3-dev \
+		make \
+		python \
+		rsync \
+	&& rm -rf /var/lib/apt/lists/ \
+	&& npm install -g coffee-script
```

Comment:
I'd like the version of this to be specified in the package.json ideally, but at the very least it needs to be specified here to avoid picking up breaking changes by accident